mythril/mythril.py

```diff
@@ -94,11 +94,12 @@ def __init__(
 
         self.mythril_dir = self._init_mythril_dir()
 
-        self.sigs = signatures.SignatureDb(
-            enable_online_lookup=self.enable_online_lookup
-        )
+        self.sigs = {}
```

Comment:
If we initialize it here then line 105/106 might throw an error right, `self.sigs.signatures_file`

Reply:
Good catch! In the old and new `SignatureDB` constructor `signatures_file` is set to None, though, so I think we can remove that part of the logging statement completely, because it won't show a path anyway

Reply:
Whoops, overlooked that `open` is called in the constructor :smile: